extensions/blocks/podcast-player/edit.js

```diff
@@ -127,6 +161,14 @@ const PodcastPlayerEdit = ( {
 			</BlockControls>
 			<InspectorControls>
 				<PanelBody title={ __( 'Podcast settings', 'jetpack' ) }>
+					<FeedURLControl
+						label={ __( 'RSS Feed URL', 'jetpack' ) }
+						url={ editedUrl || '' }
```

Comment:
```suggestion
						url={ editedUrl  }
```

As above, should we set defaults within the component?